tensorflow/compiler/xla/service/gpu/horizontal_loop_fusion.cc

```diff
@@ -463,17 +467,28 @@ Status HorizontalLoopFusionImpl::Fuse(
       TF_ASSIGN_OR_RETURN(
           HloInstruction * gep,
           MakeGetTupleElementHlo(hori_fusion_instr, total_output_id++));
-      bitcasts.push_back(computation_->AddInstruction(
-          HloInstruction::CreateBitcast(output->shape(), gep)));
+      // This pass also runs very late. So we can't expect useless
```

Comment:
Nit: rephrase? Does this run after algebraic simplifer which should clean bitcasts up?